src/pages/Sketch/components/Canvas.jsx

```diff
@@ -449,7 +452,7 @@ function Canvas() {
         types: [
           {
             description: "Doc2pen Sketch Save File",
-            accept: { "application/d2ps": [".d2ps"] },
+            accept: { "image/*": [".jpg .jpeg .png"] },
```

Comment:
@anshika208  Please dont delete important lines of code. 
'This is a braking change

Now d2ps files are saved with wrong mimeTypes and extensions

Reply:
I have changed the file format only because for opening d2ps files in our local system we need to have a definite kind of application and since the final setch was being saved in the png format that is the reason i changed the file format here